git_machete/github.py

```diff
@@ -214,6 +214,22 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                 f'`{method} {url}` request ended up in 404 response from GitHub. A valid GitHub API token is required.\n'
                 f'Provide a GitHub API token with `repo` access via one of the: {get_github_token_possible_providers()} '
                 'Visit `https://github.com/settings/tokens` to generate a new one.')  # TODO (#164): make dedicated exception here
+        elif err.code == http.HTTPStatus.TEMPORARY_REDIRECT:
+            current_repo_and_org = get_repo_and_org_names_by_id(err.headers['Location'].split('/')[4])
```

Comment:
One more thing... to make this thing really future-proof... I'd make an explicit check:
* whether `Location` header is present and non-`None`
* whether the result of `split('/')` has 4-th element
to avoid runtime exception on some unexpected HTTP response. Unlikely, but still